implementation/service_discovery/src/service_discovery_impl.cpp

```diff
@@ -2355,14 +2357,14 @@ service_discovery_impl::handle_eventgroup_subscription(
         boost::system::error_code ec;
         // TODO: Add session id
         VSOMEIP_ERROR << __func__
-                << ": Requested major version:[" << (uint32_t) _major
+                << ": Requested major version:[" << +static_cast<uint32_t>(_major)
```

Comment:
Remove typo.

```suggestion
                << ": Requested major version:[" << static_cast<uint32_t>(_major)
```

Reply:
Not a typo, see https://github.com/COVESA/vsomeip/pull/560#discussion_r1550589301